src/nft/components/sell/select/FilterSidebar.tsx

```diff
@@ -0,0 +1,229 @@
+          />
+          <Box background="lightGray" borderRadius="12" paddingTop="8" paddingBottom="8" overflow="scroll">
+            {displayCollections?.map((collection, index) => (
+              <CollectionItem
+                key={index}
+                collection={collection}
+                collectionFilters={collectionFilters}
+                setCollectionFilters={setCollectionFilters}
+              />
+            ))}
+          </Box>
+        </Column>
+      </Box>
+    </>
+  )
+}
+
+const CollectionFilterSearch = ({
+  collectionSearchText,
+  setCollectionSearchText,
+}: {
+  collectionSearchText: string
+  setCollectionSearchText: Dispatch<SetStateAction<string>>
+}) => {
+  return (
+    <Box
+      as="input"
+      borderColor={{ default: 'medGray', focus: 'genieBlue' }}
+      borderWidth="1px"
+      borderStyle="solid"
+      borderRadius="8"
+      padding="12"
+      marginLeft="0"
+      marginBottom="24"
+      backgroundColor="white"
+      fontSize="14"
+      color={{ placeholder: 'darkGray', default: 'blackBlue' }}
+      placeholder="Search collections"
+      value={collectionSearchText}
+      onChange={(e: FormEvent<HTMLInputElement>) => setCollectionSearchText(e.currentTarget.value)}
+    />
+  )
+}
+
+const CollectionItem = ({
+  collection,
+  collectionFilters,
+  setCollectionFilters,
+}: {
+  collection: WalletCollection
+  collectionFilters: Array<string>
+  setCollectionFilters: (address: string) => void
+}) => {
+  const [isCheckboxSelected, setCheckboxSelected] = useState(false)
+  const [hovered, toggleHovered] = useReducer((state) => {
+    return !state
+  }, false)
+  const isChecked = (address: string) => {
+    return collectionFilters.some((collection) => collection === address)
+  }
+  const handleCheckbox = () => {
+    setCheckboxSelected(!isCheckboxSelected)
+    setCollectionFilters(collection.address)
+  }
+  return (
+    <Row
+      cursor="pointer"
+      paddingRight="14"
+      height="44"
+      as="li"
+      background={hovered ? 'medGray' : undefined}
+      onMouseEnter={toggleHovered}
+      onMouseLeave={toggleHovered}
+      onClick={handleCheckbox}
+    >
+      <Box as="img" borderRadius="round" marginLeft="16" width="20" height="20" src={collection.image} />
+      <Box as="span" marginLeft="6" marginRight="auto" className={styles.collectionName}>
+        {collection.name}{' '}
+      </Box>
+      <Checkbox checked={isChecked(collection.address)} hovered={hovered} onChange={handleCheckbox}>
+        <Box as="span" color="darkGray" marginRight="12" marginLeft="auto">
+          {collection.count}
+        </Box>
+      </Checkbox>
+    </Row>
+  )
+}
+
+const statusArray = ['All', 'Unlisted', 'Listed']
+
+const ListStatusFilterButtons = ({
+  listFilter,
+  setListFilter,
+}: {
+  listFilter: string
+  setListFilter: (value: string) => void
+}) => {
+  return (
+    <>
+      {statusArray.map((value, index) => (
```

Comment:
tbh, I don't personally care but I know people prefer not to use the array index as the key if possible since it's written in the docs not to do so

Reply:
actually using index as a key here is fine since the array being mapped is constant so it won't change and rehydrate, but yeah you could just use the value to be legal

Reply:
I mean that's why I said I don't personally care because it's often the case the array won't change order even when fetching the data.

Reply:
I prefer the the index here over the value in this case since we don't have any explicit duplicate prevention for the array